CONTRIBUTING.rst

```diff
@@ -149,6 +149,7 @@ If you plan on doing development and testing, you will also need to install the
     pip install -r dev-requirements.txt
     pip install -r test-requirements.txt
     pip install -e .[extras]  # installs mlflow from current checkout
+    pip install -e tests/resources/mlflow-test-plugin # installs an mlflow plugin for tests
```

Comment:
```suggestion
    pip install -e tests/resources/mlflow-test-plugin # installs `mlflow-test-plugin` that is required for running MLflow plugin tests
```

Can we fix the comment a bit?

Reply:
@harupy Cheers. The thing is that this plugin is required for a lot of tests, not just plugin tests, e.g. 
https://github.com/mlflow/mlflow/blob/f7dda6e64eba16bca485b481f5ad52cfbd505187/tests/deployments/test_cli.py#L15
I'll clarify the why part in the comment though.

Reply:
You're right! Thanks for the update!